### ISSUE TITLE
New rule: Attribute groups should only contain one attribute

### ISSUE THREAD

**Issue by fourls** [integrated-application-development/sonar-delphi#66]
### Prerequisites

- [X] This rule has not already been suggested.
- [X] This should be a new rule, not an improvement to an existing rule.
- [X] This rule would be generally useful, not specific to my code or setup.

### Suggested rule title

Attribute groups should only contain one attribute

### Rule description

This rule would detect instances of multiple attributes within the same square brackets (e.g. `[Foo, Bar]`).

```delphi
type
  [Foo, Bar('test')] // Noncompliant
  TMyClass = class(TObject)
  public
    [Foo]
    [Bar('test')] // Compliant
    procedure Baz;
  end;
```

### Rationale

Having multiple attributes combined under a single square brackets is confusing, difficult to read, and error-prone. It's not immediately clear that there are multiple attributes applied, instead of a single attribute with constructor arguments.

**Comment by Cirras** [comment]
Agreed on the idea behind the rule - is there anything about this in the Embarcadero style guide?

**Comment by fourls** [comment]
Don't believe so. [This](https://docwiki.embarcadero.com/RADStudio/en/Annotating_Types_and_Type_Members) was the only reference I could find to this syntax feature, and it doesn't venture an opinion:

![image](https://github.com/integrated-application-development/sonar-delphi/assets/28214458/90cc7cc5-2b98-4bfd-b5b8-c9fde2308003)
